backend-models/service-tests/profileservice-tests.md

```diff
@@ -27,7 +25,12 @@ send {
 	$action = "challenge";
 }
 
+*********
+Service output-
+*********
 
+Save the values in encoded JSON string (for example '{"success": true, "status": "", "debug": "NONE!"}')
```

Comment:
Output should look like this [example1](https://github.com/HGustavs/LenaSYS/pull/13918/files) or like this [example2](https://github.com/HGustavs/LenaSYS/pull/13924/files)